runtime/jvmti/jvmtiThread.c

```diff
@@ -1248,22 +1271,141 @@ jvmtiGetCurrentThread(jvmtiEnv* env,
 
 #if JAVA_SPEC_VERSION >= 19
 jvmtiError JNICALL
-jvmtiSuspendAllVirtualThreads(jvmtiEnv* env,
+jvmtiSuspendAllVirtualThreads(jvmtiEnv *env,
 	jint except_count,
-	const jthread* except_list)
+	const jthread *except_list)
```

Comment:
If you're going to fix these, please fix them all (see e.g. lines 28, 30, 34-37, etc.).

Reply:
They are being fixed in https://github.com/eclipse-openj9/openj9/pull/15914/files.